pages/dkp/konvoy/2.2/choose-infrastructure/aws/air-gapped/bootstrap/index.md

```diff
@@ -11,18 +11,22 @@ enterprise: false
 
 Konvoy deploys all cluster lifecycle services to a bootstrap cluster, which deploys a workload cluster. When the workload cluster is ready, move the cluster lifecycle services to the workload cluster, after which the workload cluster manages its own lifecycle.
 
-1.  Pull the bootstrap Docker image and save it as tar.gz locally. The image version should correspond to the version of Konvoy as returned by `dkp version`:
+1.  Set an environment variable with the DKP version.
+    
+    ```bash
+    export DKP_VERSION=v2.2.0
```

Comment:
In https://github.com/mesosphere/dcos-docs-site/pull/4233#discussion_r840865116, we chose to move away from using a variable like this in the Kommander docs to be consistent with the Konvoy docs 🙈

Reply:
Its funny cause I wrote that comment too :D

Reply:
Fixed